drivers/pwm/pwm_mcux.c

```diff
@@ -137,7 +148,10 @@ static int pwm_mcux_init(const struct device *dev)
 
 	/* Disable fault sources */
 	((PWM_Type *)config->base)->SM[config->index].DISMAP[0] = 0x0000;
+#ifdef CONFIG_SOC_SERIES_IMX_RT10XX
```

Comment:
Is there a macro in the features file that defines the number of fault channels? Something like `FSL_FEATURE_PWM_NUM_FAULT_CHANNELS`?

Reply:
Good thought- `FSL_FEATURE_PWM_FAULT_CH_COUNT` looks like what I'd want. Updated the PR.